integration/run/run_test.go

```diff
@@ -1409,3 +1410,61 @@ func getPodIPFromAppName(t *testing.T, ctx context.Context, kc *crClient.WithWat
 
 	return podIP
 }
+
+func TestProjectUpdate(t *testing.T) {
+	helper.StartController(t)
+
+	rc, err := restconfig.New(scheme.Scheme)
+	if err != nil {
+		t.Fatal("error while getting rest config:", err)
+	}
+	ctx := helper.GetCTX(t)
+	c, _ := helper.ClientAndNamespace(t)
+	projectName := uuid.New().String()[:8]
+	proj1, err := c.ProjectCreate(ctx, projectName, "local", []string{"local"})
+	if err != nil {
+		t.Fatal("error while creating project:", err)
+	}
+	proj1Client, err := client.New(rc, proj1.Name, proj1.Status.Namespace)
+	if err != nil {
+		t.Fatal("error creating client for project:", err)
+	}
+	defer t.Cleanup(func() {
```

Comment:
You don't need to defer calls to `t.Cleanup`

```suggestion
    t.Cleanup(func() {
```